wiki/modding/intro.md

```diff
@@ -1,94 +1,82 @@
 # Modding Intro
-_Learn how to setup the plugin template_
+_Learn how to get started writing your own PC Mods_
 
-::: danger EDITORS NOTE
-Include new templates  
-[https://github.com/Zingabopp/BeatSaberTemplates](https://github.com/Zingabopp/BeatSaberTemplates)
+::: warning
+This guide is for making mods for the **PC** version of Beat Saber!
+
+Make sure your game is modded before trying to make a mod.  
+See instructions for [modding Beat Saber on PC.](/pc-modding.md)
 :::
 
 ## Getting Started
-Beat Saber is made in Unity 2018.4 using C# with .NET framework 4.6.
-
-Download the latest version of [Visual Studio Community.](https://visualstudio.microsoft.com/)  
-There are multiple templates that you can choose to install.
+Beat Saber is made in Unity 2019.3 using C# with .NET framework 4.6  
+You will need to download the latest version of [Visual Studio Community](https://visualstudio.microsoft.com/).
```

Comment:
Somewhere around here, perhaps put a warning that this introduction assumes you know basic to intermediate levels of C# and Unity